perfkitbenchmarker/publisher.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2014 Google Inc. All rights reserved.
+# Copyright 2015 Google Inc. All rights reserved.
```

Comment:
AFAIK there's no need to update the copyright date in existing headers. My earlier change just changed the templates, so that people get the current year when they copy them into new files (as prompted by the presubmit check).